pladdy/flask-scratch
# Dockerize

#### What's new
- Add Dockerfile to make app run in a container
- Split up logic so app can run with python or flask running it
- update Makefile

#### How to test
`make test` to verify tests pass
`make run` to verify server runs via flask
Docker...
```
make docker-build
make docker-run
curl localhost:5000/hello
```
You should get 'Hello World!' on STDOUT.
`make docker-stop` to stop container
